include/librealsense2/hpp/rs_sensor.hpp

```diff
@@ -752,5 +752,49 @@ namespace rs2
             return res;
         }
     };
-}
+
+    class debug_streaming_sensor : public sensor
```

Comment:
The extension is `DEBUG_STREAM_SENSOR`, this class is `debug_streaming_sensor`, and the file-name is `debug-streams-sensor.h`...
Let's keep them all the same, please. I suggest `debug-stream-sensor.h` and make the others the same.

Reply:
done